// match-fake-data/src/index.ts
export  const getUser = () => {
  return [
    {
      name: "Pippo",
      surname: "Falsaperla",
      level: 5,
      email: "test@test.com",
      password: "falsaperla123",
      id: 1,
    },
    {
      name: "Giulio",
      surname: "Rossi",
      level: 7,
      email: "giulio.rossi@example.com",
      password: "rossigiulio123",
      id: 2,
    },
    {
      name: "Marco",
      surname: "Bianchi",
      level: 6,
      email: "marco.bianchi@example.com",
      password: "bianchimarc0!",
      id: 3,
    },
    {
      name: "Alfredo",
      surname: "Verdi",
      level: 4,
      email: "alfredo.verdi@example.com",
      password: "verdiAlfredo!456",
      id: 4,
    },
    {
      name: "Luca",
      surname: "Neri",
      level: 8,
      email: "luca.neri@example.com",
      password: "neriluca_123",
      id: 5,
    },
    {
      name: "Luca",
      surname: "Fontana",
      level: 3,
      email: "luca.fontana@example.com",
      password: "fontana.luca2023",
      id: 6,
    },
    {
      name: "Davide",
      surname: "Marini",
      level: 5,
      email: "davide.marini@example.com",
      password: "marinidavide!",
      id: 7,
    },
    {
      name: "Paolo",
      surname: "Russo",
      level: 9,
      email: "paolo.russo@example.com",
      password: "paoloRusso99",
      id: 8,
    },
    {
      name: "Stefano",
      surname: "Gallo",
      level: 2,
      email: "stefano.gallo@example.com",
      password: "gallostefano$",
      id: 9,
    },
    {
      name: "Elios",
      surname: "Ferrari",
      level: 6,
      email: "elios.ferrari@example.com",
      password: "ferrariElios!",
      id: 10,
    },
    {
      name: "Matteo",
      surname: "Romano",
      level: 4,
      email: "matteo.romano@example.com",
      password: "romanoMatteo#",
      id: 11,
    },
    {
      name: "Francesco",
      surname: "Grandi",
      level: 7,
      email: "francesco.grandi@example.com",
      password: "grandi.Francesco!",
      id: 12,
    },
    {
      name: "Giorgio",
      surname: "De Luca",
      level: 5,
      email: "giorgio.deluca@example.com",
      password: "delucaGiorgio#1",
      id: 13,
    },
    {
      name: "Silvio",
      surname: "Rinaldi",
      level: 8,
      email: "silvio.rinaldi@example.com",
      password: "rinaldiSilvio88",
      id: 14,
    },
    {
      name: "Alessandro",
      surname: "Tosti",
      level: 3,
      email: "alessandro.tosti@example.com",
      password: "tostiAlessandro!",
      id: 15,
    },
  ];
};

export const getMatch = () => {
  return [
    {
      author: 15,
      sport: "football",
      title: "Catania vs Messina",
      imgUrl: "https://www.instituteforgovernment.org.uk/sites/default/files/styles/16_9_desktop/public/2023-03/premier-league-football-1504x846px.jpg?h=dd1b06b1&itok=3dihrnpr",
      dateTime: "2024-12-21T15:00:00",
      address: "Catania, Italy",
      difficulty: 3,
      playerAvailable: 10,
      pendingPlayers: [],
      playersIn: [],
      id: 1
    },
    {
      author: 12,
      sport: "football",
      title: "Bianchi vs Rossi",
      imgUrl: "https://www.instituteforgovernment.org.uk/sites/default/files/styles/16_9_desktop/public/2023-03/premier-league-football-1504x846px.jpg?h=dd1b06b1&itok=3dihrnpr",
      dateTime: "2024-12-15T19:30:00",
      address: "Catania, Italy",
      difficulty: 5,
      playerAvailable: 15,
      pendingPlayers: [],
      playersIn: [12],
      id: 2
    },
    {
      author: 8,
      sport: "football",
      title: "Borussia vs Bayern",
      imgUrl: "https://www.instituteforgovernment.org.uk/sites/default/files/styles/16_9_desktop/public/2023-03/premier-league-football-1504x846px.jpg?h=dd1b06b1&itok=3dihrnpr",
      dateTime: "2024-11-30T14:00:00",
      address: "La Meridiana, Catania",
      difficulty: 4,
      playerAvailable: 10,
      pendingPlayers: [],
      playersIn: [8],
      id: 3
    },
    {
      author: 15,
      sport: "football",
      title: "Futsal indor",
      imgUrl: "https://www.instituteforgovernment.org.uk/sites/default/files/styles/16_9_desktop/public/2023-03/premier-league-football-1504x846px.jpg?h=dd1b06b1&itok=3dihrnpr",
      dateTime: "2025-01-10T11:00:00",
      address: "Palazzetto di Palermo, Italy",
      difficulty: 6,
      playerAvailable: 10,
      pendingPlayers: [],
      playersIn: [15,12],
      id: 4
    },
    {
      author: 7,
      sport: "football",
      title: "Siracusa vs Noto",
      imgUrl: "https://www.instituteforgovernment.org.uk/sites/default/files/styles/16_9_desktop/public/2023-03/premier-league-football-1504x846px.jpg?h=dd1b06b1&itok=3dihrnpr",
      dateTime: "2024-12-05T18:00:00",
      address: "Siracusa, Italy",
      difficulty: 8,
      playerAvailable: 10,
      pendingPlayers: [2,3,4],
      playersIn: [7,1],
      id: 5
    }
  ];
};
